pypeit/par/pypeitpar.py

```diff
@@ -281,6 +281,11 @@ def __init__(self, trim=None, apply_gain=None, orient=None,
                                'sinusoidal along one direction, with a frequency that is ' \
                                'constant across the detector.'
 
+        defaults['use_continuum'] = False
+        dtypes['use_continuum'] = bool
+        descr['use_continuum'] = 'Subtract off the continuum level fromt he image. This correction is helpful ' \
+                                 'if you are combining arcs with multiple different lamps and exposure times. '
+
```

Comment:
I have a philosophical objection to this being in the basic image processing.  Is it possible to handle the combination of the multiple arc frames in the wavelength calibration step instead?

Expecting the answer is no, I think we should at least make this description of the parameter more ominous, to ward people off from thinking they can use it more generally.  Something like: "... For arc images from GTC/OSIRIS only."